runbooks/how-to-rerun-failed-pipeline-load.md

```diff
@@ -0,0 +1,61 @@
+# How to Re-run a Failed BFD Pipeline Load
+
+Follow this runbook to successfully reload data via BFD pipeline after a failed run.
+
+Note: If there are recently merged code changes or pending redeploys, make sure those related deployments finish before running these steps. 
```

Comment:
I think you are saying the same thing, but in particular we should call out specifically any db migrations here.

Reply:
Updated